rules/set-procmount-default/raw.rego

```diff
@@ -0,0 +1,120 @@
+package armo_builtins
+
+# Fails if container does not define the "procMount" parameter as "Default"
+deny[msga] {
+    # check if we are managing with an api-server
+    api_server := input[_]
+    is_api_server(api_server)
```

Comment:
are we checking anything on the api-server?
Is the procMountType something that can be enabled wither in the api-server or the control plane info? If so let's split to 2 cases (2 different deny policies) for each one. Similar to rule [ensure-that-the-api-server-encryption-provider-config-argument-is-set-as-appropriate](https://github.com/kubescape/regolibrary/tree/2b158108f7158b0bbe738179e948ef4f0a6c2f81/rules/ensure-that-the-api-server-encryption-provider-config-argument-is-set-as-appropriate) for e.g

Reply:
We are not checking anything on the `api-server`.
This was just a double check since the information that we need is also in the `ControlPlaneInfo`.
I think we can remove lines 5-7.

Reply:
I removed the control, since it was not necessary.